src/lib/common.ml

```diff
@@ -11,6 +11,7 @@ module Unique_name = struct
       prefix
       !x
       (Random.int 100_000)
+  let variable = create
```

Comment:
oops, I meant to start using it, because `variable` implies that the generated name will respect syntax requirements of shell variables, but I still used `Unique_name.create`.